docs/book/starter-guide/pipelines/pipelines.md

```diff
@@ -47,6 +47,17 @@ As this step has multiple outputs, we need to use the
 These names can be used to directly access the outputs of steps after running
 a pipeline, as we will see [in a later chapter](./fetching-pipelines.md).
 
+{% hint style="info" %}
+The inputs and outputs of a step are *artifacts* that are automatically tracked
+and stored by ZenML in the artifact store. Artifacts are produced by steps
+whenever your step returns an object or a value. If a step returns only a single
```

Comment:
These 3 lines are basically the artifact section I mentioned above, perhaps just extract this into a section, put it somewhere where it makes sense (`pipelines.md` is not a bad place I guess) and then we can just link to it anytime "artifact" is mentioned instead of having to explain that term again